### PR TITLE
React Router - convert name to path

### PR DESCRIPTION
**Pull request by mmmavis**

### CODE REVIEW

**Comment by mmmavis** [comment]
0 error raised from `npm test` :smile:

File: components/login/Login.jsx [hunk 1]
```diff
@@ -119,7 +119,7 @@ var Login = React.createClass({
           <div className="options">
             <ul>
               { this.renderAdminLink() ? <li>{this.renderAdminLink()}</li> : null }
-              <li><span className="fa fa-list"></span><Link to="me">Your Projects</Link></li>
+              <li><span className="fa fa-list"></span><Link to="/me">Your Projects</Link></li>
```
Comment (Pomax) [inline components/login/Login.jsx]: `to={"/me"}`

File: pages/activities/ActivitiesPage.jsx [hunk 2]
```diff
@@ -97,7 +97,7 @@ var ActivitiesPage = React.createClass({
               title="Animation on the Open Web with Fabble, Para Para, and 3D Projection Mapping"
               level="Beginner"
               link="/activities/parapara/"
-              developedBy={<div><a href="http://en.mozillafactory.org/tagged/COMOZILLA">Mozilla Factory</a> and <a href="http://twitter.com/smithisgeneric">Karen Smith</a> and remixed by <Link to="home">Mozilla Learning Networks</Link>.</div>}
+              developedBy={<div><a href="http://en.mozillafactory.org/tagged/COMOZILLA">Mozilla Factory</a> and <a href="http://twitter.com/smithisgeneric">Karen Smith</a> and remixed by <Link to="/">Mozilla Learning Networks</Link>.</div>}
```
Comment (Pomax) [inline pages/activities/ActivitiesPage.jsx]: `to={"/"}`

File: pages/clubs.jsx [hunk 3]
```diff
@@ -56,7 +56,7 @@ var WhyOrganize = (
         </div>
         <div className="col-sm-6 col-md-6 col-lg-6">
           <p>
-            <strong>You can use Mozilla’s free and educator-tested curriculum.</strong> Our <Link to="activities">curriculum</Link> is
+            <strong>You can use Mozilla’s free and educator-tested curriculum.</strong> Our <Link to="/activities">curriculum</Link> is
```
Comment (Pomax) [inline pages/clubs.jsx]: `to={"/activities"}`

File: pages/clubs.jsx [hunk 4]
```diff
@@ -255,13 +255,13 @@ var ClubsPage = React.createClass({
                onDelete={this.handleClubDelete}
                onEdit={this.handleClubEdit}/>
             </div>
-            <Link to="clubs-list" className="see-full-clubs-list">See the full list</Link>
+            <Link to="/clubs/list/" className="see-full-clubs-list">See the full list</Link>
```
Comment (Pomax) [inline pages/clubs.jsx]: `to={"/clubs/list/"}`

File: pages/events/EventsPage.jsx [hunk 5]
```diff
@@ -40,7 +40,7 @@ var EventsPage = React.createClass({
           <div className="row mp-activities-banner">
             <section>
               <div className="btn-container">
-                <Link to="maker-party-2015" className="btn btn-awsm">Get the 2015 Maker Party Activities</Link>
+                <Link to="/activities/maker-party-2015" className="btn btn-awsm">Get the 2015 Maker Party Activities</Link>
```
Comment (Pomax) [inline pages/events/EventsPage.jsx]: to={...}

File: pages/opportunities/sectionlist.jsx [hunk 6]
```diff
@@ -15,7 +15,7 @@ var sectionList = [
         description:
         (<div>
           <p>Plan an event that celebrates making and learning on the Web. Our event guides walk you through every step of the process. This role is a great fit for those who want to activate their communities through individual events.</p>
-          <p><Link to="events">Learn more about Maker Party</Link>.</p>
+          <p><Link to="/events">Learn more about Maker Party</Link>.</p>
```
Comment (Pomax) [inline pages/opportunities/sectionlist.jsx]: to={...}

File: pages/opportunities/sectionlist.jsx [hunk 7]
```diff
@@ -39,7 +39,7 @@ var sectionList = [
         description:
         (<div>
           <p>Organize a group that meets regularly to learn how to read, write and participate on the Web. Plan activities using the Mozilla Clubs curriculum. This role is perfect for educators, college students, and anyone who can meet with a group of learners regularly.</p>
-          <p><Link to="mozilla-clubs">Learn more about Mozilla Clubs</Link>.</p>
+          <p><Link to="/clubs">Learn more about Mozilla Clubs</Link>.</p>
```
Comment (Pomax) [inline pages/opportunities/sectionlist.jsx]: to={...}

File: pages/web-lit-basics.jsx [hunk 8]
```diff
@@ -26,7 +26,7 @@ var WebLitBasics = (
     <div className="col-sm-12">
       <h2>Learning Objectives</h2>
       <p>
-        The learning objectives underpinning each activity are informed by Mozilla&apos;s <Link to="web-literacy">Web
+        The learning objectives underpinning each activity are informed by Mozilla&apos;s <Link to="/activities/web-literacy">Web
```
Comment (Pomax) [inline pages/web-lit-basics.jsx]: to={...}

**Comment by Pomax** [comment]
It'll be good to use `to={...}` syntax for all the "final" `<Link>` components, so they can be easily found in the future.

**Comment by mmmavis** [comment]
Cool, I'll update the PR. Was gonna open up a followup PR but might as well just do it directly here in this PR.

**Comment by Pomax** [comment]
:sweet_potato:

**Pull request merged**